recipes/imagemagick/all/conanfile.py

```diff
@@ -327,17 +327,6 @@ def _build_configure(self):
             self, win_bash=tools.os_info.is_windows
         )
 
-        # FIXME: workaround for xorg/system adding system includes https://github.com/conan-io/conan-center-index/issues/6880
-        xft_path = os.path.join(self.build_folder, "xft.pc")
-        if os.path.exists(xft_path):
-            os.remove(xft_path)
-        if "/usr/include/uuid" in self._autotools.include_paths:
-            self._autotools.include_paths.remove("/usr/include/uuid")
```

Comment:
unfortunately I think these two lines cannot be removed yet, as `uuid` is part of xorg/system package (needed for `sm` component)

Reply:
ok, re-added